TestgenBookshopAPI/service/scripts/deploy.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/bash
+
+# deploy.sh [options] [repository]
+# Deploy the Bookshop from the specified repository.
+# You must already be logged in to the repository.
+# The default repository is the IBM icr.io/integration repository.
+
+cd $(dirname $0)/../deploy
+
+script="$(basename $0)"
+
+USAGE="Usage:
+  ${script} [options]
+  ${script} --tag <tag> [options] [repository]
+
+Common options:
+  -h, --help
+  -n, --namespace NAMESPACE
+  --jaeger-endpoint URL
+  --language LANG
+"
+
+function usage {
+  local msg="$1"
+  echo "${msg}" >&2
+  echo "${USAGE}" >&2
+  exit 2
+}
+
+tag="latest"
+
+while [[ $# > 0 ]]; do
+  case "$1" in
+  -t | --tag)
+    tag=$2
+    shift 2 || usage "Missing tag"
+    ;;
+  -n | --namespace)
+    namespace=$2
+    shift 2 || usage "Missing namespace"
+    ;;
+  --jaeger-endpoint)
+    jaeger_endpoint=$2
+    shift 2 || usage "Missing Jaeger endpoint URL"
+    ;;
+  --language)
+    language=$2
+    shift 2 || usage "Missing language"
+    ;;
+  -h | --help)
+    echo "${USAGE}"
+    exit 0
+    ;;
+  -*)
+    usage "Unrecognised option: $1"
+    ;;
+  *)
+    repository=$1
+    shift
+    ;;
+  esac
+done
+
+if [[ -z "${tag}" ]]; then
+  usage "--tag is required"
+fi
+
+tag=${tag:-latest}
```

Comment:
Since you've already checked that `tag` is not empty, this line is redundant: it was either `latest` from the start, or has been set by the `--tag` option.

Reply:
Done